pkg/kubelet/kubelet_getters.go

```diff
@@ -249,7 +249,7 @@ func (kl *Kubelet) getPodVolumePathListFromDisk(podUID types.UID) ([]string, err
 	if pathExists, pathErr := volumeutil.PathExists(podVolDir); pathErr != nil {
 		return volumes, fmt.Errorf("Error checking if path %q exists: %v", podVolDir, pathErr)
 	} else if !pathExists {
-		glog.Warningf("Warning: path %q does not exist: %q", podVolDir)
+		glog.Warningf("Warning: path %q does not exist", podVolDir)
```

Comment:
Since this is already a `glog.Warningf`, maybe it would be worth removing the `Warning:` from the beginning of the message?